### PR TITLE
Tab states maintained when navigating

### PR DESCRIPTION
**Pull request by hamzaahmed00**
Going back to the Home screen now results in the user to be in the tab from which he navigated instead of coming back to the Actions tab by default.

### CODE REVIEW

File: web/src/components/tabs/index.tsx [hunk 1]
```diff
@@ -8,13 +10,20 @@ type TabbarProps = {
...
 					<div
 						key={item.tab}
-						onClick={() => setTab(item.tab)}
+						onClick={() => {
+							setTab(item.tab)
+							h.push({
+								pathname: location.pathname,
+								search: '?activeTab=' + item.tab
+							})
+						}}
```
Comment (mudssrali) [inline web/src/components/tabs/index.tsx:26]: This component is being used in multiple places. Please pass an optional callback from parent component to push the `history`.

File: web/src/pages/home/index.tsx [hunk 2]
```diff
@@ -31,7 +32,11 @@ const TabbarContent = [
 ]
 
 export const Home = () => {
-	const [activeTab, setActiveTab] = useState(Tabs.ACTIONS)
+	const urlParams = new URLSearchParams(location.search)
+
+	const [activeTab, setActiveTab] = useState(
+		parseInt(urlParams.get('activeTab') ?? '1') ?? Tabs.ACTIONS
+	)
```
Comment (mudssrali) [inline web/src/pages/home/index.tsx:39]: Make sure, `activeTab` should be an `integer` instead of  `string`

**Review by mudssrali — CHANGES_REQUESTED** [review]

File: web/src/pages/home/index.tsx [hunk 3]
```diff
@@ -8,6 +8,8 @@ import { StatsTab } from './statistics'
 
 import { AppLayout } from 'components/Layout/appLayout'
 import { Tabbar } from 'components/tabs'
+import * as History from 'history'
```
Comment (raoaliahmad) [inline web/src/pages/home/index.tsx:11]: What's this? This is not getting used anywhere.

**Review by raoaliahmad — APPROVED** [review]
Looks good, I saw an unused import I think should remove that.

**Pull request merged**